libraries/commerce/product/selectors/product.js

```diff
@@ -200,12 +197,13 @@ const getResultByHash = createSelector(
 /**
  * Populates the product result object.
  * @param {Object} state The application state.
+ * @param {Object} props The component props.
  * @param {string} hash The result hash.
  * @param {Object} result The result.
  * @return {Object} The product result.
  */
-export const getPopulatedProductsResult = (state, hash, result) => {
-  const sort = getSortOrder(state);
+export const getPopulatedProductsResult = (state, props, hash, result) => {
```

Comment:
This is not a selector and nothing will be cached. Please change that to be a valid selector using `createSelector()`.

Reply:
I recommend against doing this. The scope of this change would be quite large. Probably is best to create a separate TD task instead.